arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -183,7 +183,6 @@
         }
       },
       "delete": {
-        "x-ms-long-running-operation": true,
```

Comment:
This is a breaking change, was this operation async earlier? Or was swagger wrong?